### DISCUSSION TITLE
Window only moveable when dragging from titlebar, not contents.

### DISCUSSION THREAD

**Discussion by saucesaft** [emilk/egui#2567] [Q&A]
So the title basically explains what im trying to do, but i need it because im developing an emulator with egui functioning as the debug gui. This emulator supports mouse click and movement, but when these 2 are used in conjunction (e.g. as a dragging movement) the whole gui window moves.

If this could help, the "screen" is rendered via a texture in an image widget as follows:
```
egui::Window::new("screen")
.show(ctx, |ui| {
    let texture = cycle.screen.display.as_ref().expect("No Texture Loaded");
    ui.image(texture, texture.size_vec2());
});
```

**Comment by NotAPenguin0** [comment]
Hi, did you find any success or a workaround? I'm looking for a similar feature.

**Reply by saucesaft** [reply]
Sorry but i didn't find a solution.

**Reply by stonerfish** [reply]
Have you seen this
 https://github.com/emilk/egui/blob/c86bfb6e67abf208dccd7e006ccd9c3675edcc2f/crates/egui_demo_app/src/backend_panel.rs#L198
